_drafts/playing-with-vsts-linux-agent-and-docker.md

```diff
@@ -42,21 +42,21 @@ Remember that if this doesn't suit your needs, you can always fall back to runni
 
 ### Getting the IP of the OrientDB container
 
-That was easy. I mean, the first Google search pointed to that [serverfault question](http://stackoverflow.com/questions/17157721/getting-a-docker-containers-ip-address-from-the-host). `docker inspect` returns information about the container, and the `--format` option allows you to specify what *portion* you want back. To only get the IP address, you can run:
+That was easy. I mean, the first Google search pointed to a [serverfault question](http://stackoverflow.com/questions/17157721/getting-a-docker-containers-ip-address-from-the-host) that explains how. `docker inspect` returns information about the container, and the `--format` option allows you to specify what *portion* you want back. To only get the IP address, you can run:
 
 {% raw %}
 `docker inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' container_name_or_id`
 {% endraw %}
 
 ### Have the tests use this IP
 
-The tests get OrientDB connection information through the [configuration model](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration) of ASP.NET Core to get information about how to connect to OrientDB. It's a provider-based model, which means you can add several sources of different types to the configuration. Providers added last can replace configuration values added by previous providers; in other words, last provider wins.
+The tests get the OrientDB connection information through the [configuration model](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration) of ASP.NET Core to get information about how to connect to OrientDB. It's a provider-based model, which means you can add several sources of different types to the configuration. Providers added last can override configuration values added by previous providers; in other words, the last provider has the highest priority.
 
-During development, all the settings are read from a JSON file. This can't work for CI since the IP address of the OrientDB container could change between runs - even though a few tries showed it wasn't the case. An option to override the hostname specified in the JSON file is to create an environment variable with the same name so it takes precedence. The configuration code then looks like:
+During development, all the settings are read from a JSON file. This can't work for CI since the IP address of the OrientDB container could change between runs - even though a few tries showed it wasn't the case. One option to override the hostname specified in the JSON file is to create an environment variable using the same name and the config setting. Environment variables are added last so they'll take precedence. The configuration code then looks like:
```

Comment:
I don't get the *and the config setting* at the end of the second sentence. Is my English failing me here?